src/components/Header/styles.scss

```diff
@@ -0,0 +1,31 @@
+.title {
+  color: #ffffff;
+  font-size: 3em;
+  font-weight: normal;
+}
+
+.searchButton {
+  background-color: #f65261;
```

Comment:
I can see this color in multiple places, we can create it in one place with scss with a name and reuse it